Dockerfile

```diff
@@ -1,7 +1,6 @@
+# -*- mode: dockerfile-mode; docker-image-name: "tailscale-dev"; -*-
 # Install dependencies only when needed
 FROM node:18-alpine AS builder
-# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
```

Comment:
libc6-compat is not needed here. We are not doing anything that needs it at this time. With my former Alpine Linux package maintainer hat on, you should prefer `gcompat` anyways.